.circleci/config.yml

```diff
@@ -165,113 +165,73 @@ jobs:
           docker-compose push web
         environment: *edge_compose_env
 
-  ingestion-beam: &ingestion_beam
-    working_directory: /root/project/ingestion-beam
+  maven:
     parameters:
       executor:
         type: executor
+      integration:
+        type: boolean
```

Comment:
Would it be more elegant to have a `scope` parameter that's an enum of `integration` and `unit` instead of a boolean `integration` param?

Reply:
sure